test/helpers/components/oneagent/daemonset.go

```diff
@@ -16,19 +16,19 @@ import (
 	"sigs.k8s.io/e2e-framework/pkg/features"
 )
 
-func WaitForDaemonset(dk dynakubev1beta2.DynaKube) features.Func {
+func WaitForDaemonset(dk dynakubev1beta3.DynaKube) features.Func {
 	return helpers.ToFeatureFunc(daemonset.WaitFor(dk.OneAgentDaemonsetName(), dk.Namespace), true)
 }
 
 func WaitForDaemonsetV1Beta1(dk dynakubev1beta1.DynaKube) features.Func {
```

Comment:
still necessary? wouldn't `WaitFromDaemonSetPrevDk` more inline with the other helpers?

Reply:
sure, but current problem which version we want to test.

Reply:
can't we just pass the value of the `dk.OneAgentDaemonsetName` method into the function instead of duplicating it for every dynakube version?

Reply:
fixed in 2cbbd90f